packages/vue/src/components/basic/SelectedFilters.jsx

```diff
@@ -67,7 +71,43 @@ const SelectedFilters = {
 			this.$emit('clear', component, value);
 		},
 		clearValues() {
-			this.clearValuesAction();
+			const { resetToDefault, componentProps, selectedValues } = this;
+			if (resetToDefault) {
```

Comment:
For readability purposes you can deconstruct these conditions into separate methods with a single purpose.

Reply:
Agreed! 👍  clearValuesToDefault()